libvast/vast/system/actors.hpp

```diff
@@ -201,16 +201,15 @@ using meta_index_actor = typed_actor_fwd<
   caf::replies_to<atom::merge, std::shared_ptr<std::map<
                                  uuid, partition_synopsis>>>::with<atom::ok>,
   // Merge a single partition synopsis.
-  caf::replies_to<atom::merge, uuid, std::shared_ptr<partition_synopsis>>::with< //
-    atom::ok>,
+  caf::replies_to<atom::merge, uuid,
+                  std::shared_ptr<partition_synopsis>>::with<atom::ok>,
   // Erase a single partition synopsis.
   caf::replies_to<atom::erase, uuid>::with<atom::ok>,
   // Atomically remove one and merge another partition synopsis
   caf::replies_to<atom::replace, uuid, uuid,
                   std::shared_ptr<partition_synopsis>>::with<atom::ok>,
   // Evaluate the expression.
-  caf::replies_to<atom::candidates, vast::expression,
-                  vast::ids>::with<std::vector<vast::uuid>>>
+  caf::replies_to<atom::candidates, vast::query>::with<std::vector<vast::uuid>>>
```

Comment:
The comment above is outdated, as this now evaluates a query rather than an expression. Also we could probably say "return candidate partitions for query" rather than "evaluate expression/query", as that is much easier to understand.